This allows us to get some function from math.h that other platforms are
getting elsewhere.
Can send upstream with ifdef Solaris statement. 

--- mozilla-esr31/content/media/webaudio/blink/DenormalDisabler.h.orig	2015-06-04 17:42:51.279492311 -0700
+++ mozilla-esr31/content/media/webaudio/blink/DenormalDisabler.h	2015-06-04 17:42:51.288558769 -0700
@@ -27,6 +27,7 @@
 
 #define _USE_MATH_DEFINES
 #include <cmath>
+#include <math.h>
 #include <float.h>
 
 namespace WebCore {
--- mozilla-esr31/content/media/webaudio/ThreeDPoint.h.orig	2015-06-04 17:42:51.286104792 -0700
+++ mozilla-esr31/content/media/webaudio/ThreeDPoint.h	2015-06-04 17:42:51.288664521 -0700
@@ -8,6 +8,7 @@
 #define ThreeDPoint_h_
 
 #include <cmath>
+#include <math.h>
 #include <algorithm>
 
 namespace mozilla {
